pkg/config/config.go

```diff
@@ -192,7 +192,8 @@ type Config struct {
 		Type                   string `mapstructure:"type" validate:"required"`
 		DefaultNamespacePrefix string `mapstructure:"default_namespace_prefix"`
 		Local                  *struct {
-			Path string `mapstructure:"path"`
+			Path                    string   `mapstructure:"path"`
```

Comment:
Since we can't really guarantee this will work - I suggest making import on local adapter configurable - with the default value disabled.
In addition to that we should document the list of filesystems tested to be working correctly and any caveats we have for using import on local block adapter

Reply:
Agree - will apply in a different PR as we also need to remove the warning on startup and in the UI